proxies/php/nginx/nginx.conf

```diff
@@ -0,0 +1,35 @@
+server {
+    listen 3000 default_server;
+
+    server_name _;
+    root /var/www/html;
+
+    # Match only /client to create new clients in the proxy manager.
+    location = /client {
+        proxy_pass http://127.0.0.1:8000$request_uri;
+    }
+
+    location ~ /close/.* {
+            proxy_pass http://127.0.0.1:8000$request_uri;
+        }
+    # Match the /<clientId>/v1/.* pattern to route to the correct socket proxy
+    location ~ ^/(.*)(/v1/.*)$ {
+        proxy_pass http://unix:/tmp/$1.sock:$2;
+    }
```

Comment:
I think the above block is unused? The PHP SDK seems to use the Unix socket directly, and removing this had no effect on the tests that I could see.

Reply:
This is to allow you to take a test locally and run it against the container if you breakpoint the test harness.

It's a thing I was using during development, but may be useful otherwise?